File: apps/web/backend/app/Models/UserNotificationConfig.php
<?php
namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class UserNotificationConfig extends Model
{
    protected $table = 'USER_NOTIFICATION_CONFIG';
    protected $primaryKey = 'ID';
    public $timestamps = false;

    protected $fillable = ['USER_ID', 'PUSH', 'EMAIL', 'SMS'];
}

File: apps/dotnet/Simex/Simex/Models/UserNotificationConfig.cs
using System;
using System.Collections.Generic;

namespace Simex.Models;

public partial class UserNotificationConfig
{
    public int Id { get; set; }

    public int? UserId { get; set; }

    public bool Push { get; set; }

    public bool Email { get; set; }

    public bool Sms { get; set; }

    public virtual User? User { get; set; }
}
